### PR TITLE
Add recipe for QuPy

### PR DESCRIPTION
**Pull request by ken-nakanishi**

### CODE REVIEW

File: recipes/qupy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
```
Comment (jentjr) [inline recipes/qupy/meta.yaml]: Please pin compatible numpy
```suggestion
    - {{ pin_compatible('numpy') }}
```

Comment (ken-nakanishi) [reply recipes/qupy/meta.yaml]: Thanks!

**Comment by jentjr** [comment]
Sorry! I reviewed the recipe too quickly, and did not catch that numpy was not in the host section. If you are not linking against numpy, then I believe it is fine to have numpy just in the run section without pinning. 
[https://conda-forge.org/docs/meta.html#building-against-numpy](url)

**Comment by ken-nakanishi** [comment]
Thanks! I unpinned numpy.

**Comment by jentjr** [comment]
Looks like Travis failed, but not because of your recipe. Try triggering a rebuild. [https://github.com/conda-forge/staged-recipes#11-some-checks-failed-but-it-wasnt-my-recipe-how-do-i-trigger-a-rebuild](url)

**Comment by jentjr** [comment]
@conda-forge/staged-recipes This is my first review, but everything looks good to me. Please review for merging. Thanks!

**Comment by ken-nakanishi** [comment]
Thanks @jentjr !

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**